sotodlib/site_pipeline/preprocess_tod.py

```diff
@@ -42,16 +42,19 @@ def _get_preprocess_context(configs, context=None):
     return configs, context
 
 def _get_groups(obs_id, configs, context):
-    group_by = configs['subobs'].get('use', 'detset')
-    if group_by.startswith('dets:'):
-        group_by = group_by.split(':',1)[1]
+    group_by = np.atleast_1d(configs['subobs'].get('use', 'detset'))
```

Comment:
will this fail if "use" is None? TBH I'm fine with it not working if use is None but maybe want a more useful error message than I think this will make.